src/main/java/org/spongepowered/api/command/args/GenericArguments.java

```diff
@@ -921,22 +922,35 @@ public static CommandElement bool(Text key) {
 
     private static class EnumValueElement<T extends Enum<T>> extends PatternMatchingCommandElement {
         private final Class<T> type;
+        private final Map<String, T> values;
```

Comment:
Nope, its not.
[`Collectors#toMap(...)`](http://grepcode.com/file/repository.grepcode.com/java/root/jdk/openjdk/8-b132/java/util/stream/Collectors.java#1267)